HLTrigger/Configuration/test/cmsDriver.csh

```diff
@@ -13,13 +13,13 @@ rehash
 #
 # gen sim input files for Monte-Carlo tests
 #   InputGenSimGRun0 = /store/relval/CMSSW_8_0_11/RelValProdTTbar/GEN-SIM/80X_mcRun1_realistic_v4-v1/10000/06A6C86B-C634-E611-93A5-0CC47A74525A.root
-set InputGenSimGRun0 = /store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/GEN-SIM/CMSSW_8/06A6C86B-C634-E611-93A5-0CC47A74525A.root
+set InputGenSimGRun0 = root://eoscms.cern.ch//eos/cms/store/group/dpg_trigger/comm_trigger/TriggerStudiesGroup/STORM/GEN-SIM/CMSSW_8/06A6C86B-C634-E611-93A5-0CC47A74525A.root
```

Comment:
I've checked it here https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-16534/16292/addOnTests/ and I see that the GEN-SIM input files are generated in each test. 
Therefore, the "InputGenSimGRun*" files are not used in the integration build tests.